smt_PCB_files/README.txt

```diff
@@ -0,0 +1,3 @@
+How to use：
+
+At editor, open the document via: Top menu - File - Open - EasyEDA... , and select the json file, then open it at the editor, you can save it into a project.
```

Comment:
what editor? [EasyADA](https://easyeda.com/)? if so, please directly add a link to it